src/publisher.cpp

```diff
@@ -85,8 +101,6 @@ void Publisher::advertise()
     return;
   if ( topic_.isEmpty() )
     return;
-  if ( queue_size_ == 0 )
-    return;
```

Comment:
Would we want to check `qos_` here in some way?

Reply:
No, I don't think that check was necessary in the first place.